src/BugsnagServiceProvider.php

```diff
@@ -70,6 +70,10 @@ public function register()
             $options = ['base_uri' => isset($config['endpoint']) ? $config['endpoint'] : Client::ENDPOINT];
 
             if (isset($config['proxy']) && $config['proxy']) {
+                if (isset($config['proxy']['http']) && php_sapi_name() != 'cli') {
```

Comment:
Don't allow the option through if we're on a vulnerable system because it's like it was set by an env variable.